app/code/Magento/Cms/Controller/Adminhtml/Wysiwyg/Images/OnInsert.php

```diff
@@ -34,26 +42,10 @@ public function __construct(
      */
     public function execute()
     {
-        $imagesHelper = $this->_objectManager->get(\Magento\Cms\Helper\Wysiwyg\Images::class);
         $request = $this->getRequest();
 
-        $storeId = $request->getParam('store');
-
-        $filename = $request->getParam('filename');
-        $filename = $imagesHelper->idDecode($filename);
-
-        $asIs = $request->getParam('as_is');
-
-        $forceStaticPath = $request->getParam('force_static_path');
-
-        $this->_objectManager->get(\Magento\Catalog\Helper\Data::class)->setStoreId($storeId);
-        $imagesHelper->setStoreId($storeId);
-
-        if ($forceStaticPath) {
-            $image = parse_url($imagesHelper->getCurrentUrl() . $filename, PHP_URL_PATH);
-        } else {
-            $image = $imagesHelper->getImageHtmlDeclaration($filename, $asIs);
-        }
+        /** @var \Magento\Cms\Model\Wysiwyg\Images\PrepareImage $image */
```

Comment:
Incorrect and not necessary annotation. `$image` is a string and that is already defined as a return type of called method